### DISCUSSION TITLE
Project: start/end time

### DISCUSSION THREAD

**Discussion by idemus** [kimai/kimai#3673] [Questions & Answers]
Hello,

in system settings start and end time or duration could bei configuered globally.
Is there a chance to configuere it per project?
For billable reasons only duration is relevant, for working time the exactly start/end time gets more into account.

Best regards
Klaus

**Comment by kevinpapst** [comment]
I read this 3 times, still do not understand what you are trying to say. Can you rephrase please.

**Reply by idemus** [reply]
For some customers I need only duration input field, for others start and end time. As far as I can see there is only the possibility to customize duration or start/end time for ALL timesheets. It can't be distinguished so activity/project A gets duration field only, activity/project B gets start/end time input fields without duration, and so on.

**Reply by kevinpapst** [reply]
Aha, understood. Thanks for the clarification.
I have no plans to add such a feature.

You could build a custom solution with a very simple frontend using the API.
